doc/getting-started.md

```diff
@@ -8,31 +8,46 @@ This short introduction will teach you the basics of using your Almond.
 
 ## What is Almond?
 
-Almond is the magic virtual assistant that you access through your phone. She can
-help you configure your things, execute actions on them, install apps based on
-your description of the behavior you want.
+Almond is the open virtual assistant that you can access through both of 
+your phone and the [web](/me). It can help you configure your devices, accounts, 
+and public services, retrieve data from them, and execute actions on them.
+It also allows you to easily share your data with your friends and families 
+with control in your hand. 
+Oh, by the way, it's open source and it protects your privacy!
 
 ## What is Thingpedia?
 
-Thingpedia is a research project to collect interfaces and apps for the
-Almond virtual assistant. It lives [here](https://thingpedia.stanford.edu/),
-and includes an installation of Almond as a web service available free of charge
-to the research community.
+Almond builds on top of [Thingpedia](https://thingpedia.stanford.edu/), 
+an encyclopedia of applications for the Internet services and IoT devices. 
+Just like how Wikipedia stores knowledge about the world, 
+Thingpedia stores knowledge about devices in the world. 
+Wikipedia is organized around articles; 
+Thingpedia is organized around devices, such as Twitter, a light bulb, or a thermostat.
+It creates an interoperable web of devices and let users program them in 
+natural language. 
 
 ## What can I do with Almond?
 
-Almond will execute commands that use things. Therefore, to have it do anything,
-you must associate her with your things, and then give her a command. A command
-can be an action (eg. "post on twitter"), a time based rule (eg. "show a popup every 10 minutes")
-or a standing query (eg. "monitor xkcd").
+Almond draws the power from Thingpedia. 
+It supports simple commands including 
+actions (eg. `post on twitter`),
+time-based rules (eg. `tweet every hour`),
+one-time queries (eg. `search tweets`),
+and standing queries (eg. `monitor new tweets`).
+On top of one-time queries and standing queries, filters can be used to further refine 
+the commands (eg. `search tweets from bob`, `monitor new tweets about almond`).
+
+Almond also supports compound commands which interoperate functions from different devices.
+For example, you can ask Almond to `send me a slack message if bob tweets`, 
+`create a meme and tweet it`.
 
 ## Step-by-step example: Twitter to Almond
 
 This example will guide you through filtering your Twitter feed and redirect
-to Almond. At the end of the example, she will tell you about any tweet in your
-stream containing the hashtag "sabrina".
+to Almond. At the end of the example, it will tell you about any tweet in your
+stream containing the hashtag "almondapp".
```

Comment:
Having noticed that there are way to many almonds out there, maybe we should choose a different hashtag that is likely to pull up garbage? How about
```suggestion
stream containing the hashtag "#stanford".
```